src/app/public/src/modules/table-of-contents/table-of-contents-wrapper.component.spec.ts

```diff
@@ -100,7 +100,8 @@ describe('Table of Contents Wrapper Component', () => {
         { provide: StacheRouteService, useValue: mockStacheRouteService },
         { provide: SkyAppResourcesService, useValue: mockSkyAppResourcesService },
         { provide: StacheWindowRef, useValue: mockWindowService },
-        { provide: Renderer2, useValue: mockRenderer }
+        { provide: Renderer2, useValue: mockRenderer },
+        StacheOmnibarAdapterService
```

Comment:
We want to use a Mock here so we can avoid cross testing pollution as much as possible.